frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsAndSoil.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ *
+ * THIS IS A SKELETON!
+ * THIS IS STILL TO BE IMPLEMENTED!
+ *
+ * @summary Farm Information Input Module
+ * @description A module with input fields to be used in a Form Module
+ * @author @GDamaso
+ */
+import Button from '@Commons/Button/Button.tsx';
+import InputModuleInterface from 'src/Interface/InputModuleinterface';
+import { faWheatAwn } from '@fortawesome/free-solid-svg-icons';
+import { StyledFarmInfo, StyledButtonContainer } from './FieldsAndSoil.style';
+
+const FarmInfoComponent = () => (
+  <StyledFarmInfo>
+    <label htmlFor="farmName">
+      Name
+      <br />
+      <input
+        type="text"
+        name="farmName"
+      />
```

Comment:
Hardcoding linebreaks like this isn't good practice and is mostly frowned upon :) 
You could always create a common div component with a label already in it, with a `for` prop or something similar.
The CSS could be styled with the Linebreaks appropriately and you get tighter control over your work